tequilapi_spec.json

```diff
@@ -0,0 +1,202 @@
+{
+  "swagger": "2.0",
+  "paths": {
+    "/identities": {
+      "post": {
+        "description": "Create responds with new identity",
+        "tags": [
+          "Identity"
+        ],
+        "operationId": "createIdentity",
+        "responses": {
+          "200": {
+            "description": "identityDto",
+            "schema": {
+              "$ref": "#/definitions/identityDto"
+            }
+          },
+          "400": {
+            "description": "errorMessage",
+            "schema": {
+              "$ref": "#/definitions/errorMessage"
+            }
+          },
+          "422": {
+            "$ref": "#/responses/validationErrorMessage"
+          },
+          "500": {
+            "description": "errorMessage",
+            "schema": {
+              "$ref": "#/definitions/errorMessage"
+            }
+          }
+        }
+      }
+    },
+    "/identities/:id/registration": {
+      "put": {
+        "description": "Registers provided identity",
+        "produces": [
+          "application/json"
+        ],
+        "schemes": [
+          "http"
+        ],
+        "tags": [
+          "Identity"
+        ],
+        "operationId": "registerIdentity",
+        "parameters": [
+          {
+            "type": "string",
+            "x-go-name": "Id",
+            "description": "Identity to pass",
+            "name": "id",
+            "in": "path",
+            "required": true
+          }
+        ],
+        "responses": {
+          "202": {},
+          "400": {
+            "description": "errorMessage",
+            "schema": {
+              "$ref": "#/definitions/errorMessage"
+            }
+          },
+          "500": {
+            "description": "errorMessage",
+            "schema": {
+              "$ref": "#/definitions/errorMessage"
+            }
+          },
+          "501": {
+            "description": "errorMessage",
+            "schema": {
+              "$ref": "#/definitions/errorMessage"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "FieldErrorMap": {
+      "type": "object",
+      "x-go-package": "github.com/mysterium/node/tequilapi/validation"
+    },
+    "IdentityURLParams": {
```

Comment:
Is this definition used anywhere?

Reply:
IdentityURLParams is never used in go code, it is only for swagger to declare a parameter